src/pgwire/src/protocol.rs

```diff
@@ -457,6 +469,7 @@ where
 
         let result = match self.adapter_client.execute(EMPTY_PORTAL.to_string()).await {
             Ok(response) => {
+                self.metrics.query_status("success").inc();
```

Comment:
I have a preference that any query measuring we do happen directly on `adapter_client` (`SessionClient`) because 1) you wouldn't have to duplicate metric counting like in the current diff and 2) it'll also just work for our http sql interface.

Reply:
yeah, this makes sense - i'll see what moving it in there looks like